solidity/test/TestOperatorParams.js

```diff
@@ -1,6 +1,8 @@
-const OperatorParamsStub = artifacts.require('./stubs/OperatorParamsStub.sol');
+const {contract, web3} = require("@openzeppelin/test-environment")
+var assert = require('chai').assert
+const OperatorParamsStub = contract.fromArtifact('OperatorParamsStub');
 
-contract('OperatorParamsStub', (accounts) => {
+describe('OperatorParamsStub', () => {
```

Comment:
This one should describe `OperatorParams`, the fact we need to test through the stub is an implementation detail.